File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/01-lab/06.building.js
function building(input) {

    const floorsCount = Number(input[0]);
    const roomsCountPerFloor = Number(input[1]);

    for (let f = floorsCount; f >= 1; f--) {
        let printLine = "";
        for (let r = 0; r < roomsCountPerFloor; r++) {
            if (f === floorsCount) {
                printLine += `L${f}${r} `;
            } else if (f % 2 !== 0) {
                printLine += `A${f}${r} `;
            } else {
                printLine += `O${f}${r} `;
            }
        }
        console.log(printLine);
    }
}
building(["9", "5"]);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/01.lab/06.building.py
floors_count = int(input())
rooms_count_per_floor = int(input())

for f in range(floors_count, 0, -1):
    printLine = ""
    for r in range(rooms_count_per_floor):
        if f == floors_count:
            printLine += f'L{f}{r} '
        elif f % 2 != 0:
            printLine += f'A{f}{r} '
        else:
            printLine += f'O{f}{r} '
            
    print(printLine)
    